content/en/dynamic_instrumentation/symdb/java.md

```diff
@@ -1,31 +1,28 @@
 ---
-title: Enable Symbol Database for Java
+title: Enable autocomplete and search for Java
 kind: Documentation
 is_beta: true
-private: true
+private: false
 code_lang: java
 type: multi-code-lang
 code_lang_weight: 10
 ---
 
-{{< beta-callout-private url="https://forms.gle/UG9EELAy8Li6z2jW8" >}}
-Interested in an improved user experience when creating Dynamic Instrumentation probes? Join the Symbol Database private beta here.
-{{< /beta-callout-private >}}
-
-[Symbol Database][6] is a feature of Dynamic Instrumentation in private beta.
+{{< beta-callout url="#" btn_hidden="true" >}}
+Autocomplete and search are in public beta.
+{{< /beta-callout >}}
 
 ## Requirements
 
 - [Dynamic Instrumentation][1] is enabled for your service.
-- Tracing library [`dd-trace-java`][6] 1.25.0 or higher is installed.
+- Tracing library [`dd-trace-java`][6] 1.34.0 or higher is installed.
 
 ## Installation
 
-To run your service with Dynamic Instrumentation enabled, and additionally enable Symbol Database uploads:
+Run your service with Dynamic Instrumentation enabled, and additionally enable autocomplete and search:
 
-1. Set the `-Ddd.symbol.database.upload.enabled` flag or the `DD_SYMBOL_DATABASE_UPLOAD_ENABLED` environment variable to `true`. 
-2. Set the `-Ddd.symbol.database.includes` flag or the `DD_SYMBOL_DATABASE_INCLUDES` environment variable to your package prefix (for example, `com.datadoghq`). The setting expects a comma-separated list, so you can add multiple prefixes. 
-3. Specify the `dd.service` and `dd.version` [Unified Service Tags][5].
+1. Set the `-Ddd.symbol.database.upload.enabled` flag or the `DD_SYMBOL_DATABASE_UPLOAD_ENABLED` environment variable to `true`.
```

Comment:
should we just spell it out like `-Ddd.symbol.database.upload.enabled=true` flag or the `DD_SYMBOL_DATABASE_UPLOAD_ENABLED=true` instead of write that they should do it?
make copy paste much easier and less chance of typos.